code/modules/mob/living/carbon/inventory.dm

```diff
@@ -74,6 +74,9 @@
 		if(ITEM_SLOT_BACKPACK)
 			if(!back || !SEND_SIGNAL(back, COMSIG_TRY_STORAGE_INSERT, I, src, TRUE))
 				not_handled = TRUE
+		if(ITEM_SLOT_LHANDSTORE)
+			if(!held_items[LEFT_HANDS] || !SEND_SIGNAL(held_items[LEFT_HANDS], COMSIG_TRY_STORAGE_INSERT, I, src, TRUE))

```

Comment:
This is giving a lint warning because held_items has no type hint, nor should it since it contains nulls.

```dm
	/**
	  * list of items held in hands
	  *
	  * len = number of hands, eg: 2 nulls is 2 empty hands, 1 item and 1 null is 1 full hand
	  * and 1 empty hand.
	  *
	  * NB: contains nulls!
	  *
	  * held_items[active_hand_index] is the actively held item, but please use
	  * get_active_held_item() instead, because OOP
	  */
	var/list/held_items = list()
```
So use the proc to get the held item